style.css

```diff
@@ -0,0 +1,134 @@
+.body {
+    max-width: 1920px;
+    min-width: 375px;
+    margin: 0 auto;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)